restore-sql-server-transaction-logs/README.md

```diff
@@ -0,0 +1,319 @@
+
+This repository also contains a powershell script for regularly uploading new files to cloud storage called upload-script.ps1, existing in the scheduled-upload folder. This provides an automated way of uploading the backup files logs to cloud storage.
+
+The function must have defined a set of environment variables. Details about them are described below, in the constraints section.
+
+
+## Setup and configuration - Cloud Function
+
+
+For the operations in this section below you need the following permissions:
+
+                cloudsql.instances.get
+                cloudsql.instances.list
+                storage.buckets.create
+                storage.buckets.get
+                storage.buckets.getIamPolicy
+                storage.buckets.setIamPolicy
+                storage.buckets.update
+                storage.buckets.list
+                iam.roles.create
+                iam.roles.delete
+                iam.roles.get
+                iam.roles.list
+                iam.roles.undelete
+                iam.roles.update
+                iam.serviceAccounts.create
+                iam.serviceAccounts.get
+                iam.serviceAccounts.list     
+                resourcemanager.projects.list
+                resourcemanager.projects.get
+                resourcemanager.projects.getIamPolicy
+                resourcemanager.projects.setIamPolicy
+
+and member of the following predefined roles:
+                
+                roles/cloudfunctions.developer
+
+or, your user must be member of the following predefined roles:
+
+                roles/cloudsql.editor
+                roles/storage.admin
+                roles/iam.roleAdmin
+                roles/iam.serviceAccountCreator
+                roles/resourcemanager.projectIamAdmin
+                roles/cloudfunctions.developer
+
+
+1. Create a GCS bucket to upload your transaction log backup files:
+        
+        export PROJECT_ID=`gcloud config get-value project`
+
+        gcloud storage buckets create gs://<BUCKET_NAME> \
+        --project= ${PROJECT_ID} \
+        --location=<BUCKET_LOCATION> \
+        --public-access-prevention
+
+1. Use the gcloud describe command to get the service account information of your Cloud SQL Instance
+
+        gcloud sql instances describe <CLOUD_SQL_INSTANCE_NAME>
+
+Copy the value of the serviceAccountEmailAddress field. It should be something in the form of p******@${PROJECT_ID}.iam.gserviceaccount.com.
+
+1. Grant objectViewer rights for the CloudSQL service account on the bucket you just created:
+
+        gsutil iam ch serviceAccount:<service-account-email-address>@gcp-sa-cloud-sql.iam.gserviceaccount.com:legacyBucketReader,objectViewer gs://<BUCKET_NAME>

```

Comment:
I think we should use the `gcloud storage` command instead of `gsutil`. See https://cloud.google.com/storage/docs/gsutil#should-you-use

Reply:
Thanks! Replaced the command.

Reply:
LGTM